CMakeLists.txt

```diff
@@ -406,6 +406,27 @@ if(ENABLE_TESTS OR ENABLE_BENCHMARKS)
     target_include_directories(hacl_cpu_features PUBLIC ${PROJECT_SOURCE_DIR}/cpu-features/include)
 endif(ENABLE_TESTS OR ENABLE_BENCHMARKS)
 
+# Add ecckiila for benchmarks
+if(ENABLE_BENCHMARKS)
+    include(${PROJECT_SOURCE_DIR}/third-party/ecckiila/config.cmake)
+    add_library(ecckiila OBJECT ${SOURCES_ecckiila})
+endif(ENABLE_TESTS OR ENABLE_BENCHMARKS)
+
+# Add blake2 for benchmarks
+if(ENABLE_TESTS OR ENABLE_BENCHMARKS)
```

Comment:
Shouldn't all of these be only `ENABLE_BENCHMARKS`?

Reply:
I guess we could test them when benchmarking. I wondered if we want to do some cross-testing.